src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -106,7 +106,7 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Boolean updatedHasPaid = editPersonDescriptor.getHasPaid().orElse(personToEdit.getHasPaid());
 
         return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedBirthday,
-                updatedTags, updatedHasPaid);
+                updatedTags, updatedHasPaid, personToEdit.getFrequency());
```

Comment:
Not sure if this should follow the convention as with rest of the tags? i.e. updatedFrequency?